.changeset/tame-jokes-do.md

```diff
@@ -0,0 +1,15 @@
+---
+'@backstage/plugin-scaffolder-backend': minor
+---
+
+Improved the `parseEntityRef` Scaffolder filter by introducing the ability for users to provide default kind and/or namespace values. The filter now takes
```

Comment:
Think that we can simplify this changeset a little bit, but add the examples etc here to it's on documentation somewhere.

Maybe adding a `Built in Filters` section to this part? https://backstage.io/docs/features/software-templates/writing-templates

Reply:
Added docs for the others filters too, let me know if that's ok.